### PR TITLE
enhance: kafka data refference session website

### PR DESCRIPTION
**Pull request by indraprasetya154**
## Overview
- enhance: kafka data refference session website

cc: @jabardigitalservice/jds-backend 

## Evidence
title: - enhance: kafka data refference session website
project: SIKD
participants: @indraprasetya154 @samudra-ajri @fajarhikmal214

### CODE REVIEW

File: src/app/Models/Group.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Group extends Model
+{
+    use HasFactory;
+
+    protected $connection = 'sikdweb';
```
Comment (fajarhikmal214) [inline src/app/Models/Group.php:12]: is that not problem raw data like this ? or possible using env ?

Comment (indraprasetya154) [reply src/app/Models/Group.php:12]: FYI this $connection value has been set hardcode on all model, but nice idea to set connection on env, will be fix later after this PR 😄 @samudra-ajri

**Review by samudra-ajri — APPROVED** [review]

File: src/app/Models/Group.php (hunk 1 above)
Comment (fajarhikmal214) [reply src/app/Models/Group.php:12]: all rigth

**Review by fajarhikmal214 — APPROVED** [review]

File: src/app/Models/Group.php (hunk 1 above)
Comment (indraprasetya154) [reply src/app/Models/Group.php:12]: after i see code again, this connection is set on config/database.php https://github.com/jabardigitalservice/office-services/blob/60b972d84d8f4d038446a89ba017feb9304f313e/src/config/database.php#L66

is still need set on env or not? need your suggestion ya 🐱

**Pull request merged**